regparser/notice/compiler.py

```diff
@@ -79,12 +79,14 @@ def make_root_sortable(label, node_type):
     need to be sorted correctly. This returns a tuple to help
     sort these first level nodes. """
 
-    if node_type == Node.SUBPART or node_type == Node.EMPTYPART:
+    if node_type == Node.EMPTYPART:
         return (0, label[-1])
-    elif node_type == Node.APPENDIX:
+    elif node_type == Node.SUBPART:
```

Comment:
There was an issue with X in which having subpart and emptypart at the same level of precedence generated incorrect output. So now emptypart comes before subpart.